pulp_ansible/app/migrations/0006_alter_collection_version_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2019-08-02 22:26
```

Comment:
How would you feel about combining these two migrations since they are for one feature/PR?